ecosystem/sep-0038.md

```diff
@@ -174,13 +195,18 @@ This endpoint can be used to fetch the [indicative](https://www.investopedia.com
 
 These prices are indicative. The actual price will be calculated at conversion time once the Anchor receives the funds from a User.
 
-The prices returned include any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, or other factors.
+The prices returned include any margin that the provider may keep as a service fee, and this margin may vary depending on the directional flow of funds, amount, delivery method, country code, or other factors.
 
 #### Request
 
 Name | Type | Description
 -----|------|------------
 `sell_asset` | string | The asset you want to sell, using the [Asset Identification Format](#asset-identification-format).
+`sell_amount` | string | The amount of `sell_asset` the client would exchange for each of the `buy_assets`.
+`sell_delivery_method` | string | (optional) One of the `name` values specified by the `sell_delivery_methods` array for the associated asset returned from [`GET /info`](#info). Must be provided if the user is making an off-chain deposit to the anchor.
+`buy_delivery_method` | string | (optional) One of the `name` values specified by the `buy_delivery_methods` array for the associated asset returned from [`GET /info`](#info). Must be provided if the user intends to receive an off-chain deposit from the anchor.
```

Comment:
Same thing here, and for the attributes on the `/price` endpoint.

```suggestion
`buy_delivery_method` | string | (optional) One of the `name` values specified by the `buy_delivery_methods` array for the associated asset returned from [`GET /info`](#info). Must be provided if the user intends to receive an off-chain asset from the anchor.
```

Reply:
Fixed in https://github.com/stellar/stellar-protocol/pull/989/commits/4256e69a463f879755c70b35e3032e28a3643391